cortex/quickflat/view.py

```diff
@@ -341,14 +341,23 @@ def make_svg(fname, braindata, with_labels=False, with_curvature=True, layers=['
     height : int
         Height of PNG in pixels
     overlay_file : str
-    	Custom ROI overlays file to use
-
+        Custom ROI overlays file to use
+    with_dropout : bool or Dataview
+        If True or a cortex.Dataview object, hatches will be overlaid on top of the
+        flatmap to indicate areas with dropout. If set to True, the dropout areas will
+        be estimated from the intensity of the reference image. If set to a
+        cortex.Dataview object, values in the dataset will be considered dropout areas.
+        The transparency of the hatches is proportional to the intensity of the values
+        in the dropout dataset.
     """
     fp = io.BytesIO()
-    from matplotlib.pylab import imsave
+    from matplotlib.pyplot import imsave
 
     ## Render PNG file & retrieve image data
     arr, extents = make_flatmap_image(braindata, height=height, **kwargs)
+    # Set nans to alpha = 0. to enable transparency when saving as PNG
```

Comment:
Would it make sense to put the nan alpha masking inside `make_flatmap_image`?

Reply:
yeah I thought about it, but I wasn't sure if this would have unintended consequences when plotting with matplotlib...